app/package.json

```diff
@@ -10,13 +10,13 @@
   },
   "scripts": {
     "build": "next build",
-    "dev:next": "next dev",
+    "dev:next": "TZ=UTC next dev",
```

Comment:
There may be a more elegant way to do this. For some reason Kysely gets all confused about timezones if we don't inform it that our dates are stored in UTC time.

Reply:
Huh. What are the symptoms of it getting confused about timezones? Does it return timestamps in the wrong timezone so they display wrong to the user?

Reply:
Kysely very helpfully converts the times from PT to UTC for us (that is, it adds 7 hours to whatever time is stored in the database)